cms/grading/tasktypes/Batch.py

```diff
@@ -262,6 +265,7 @@ def evaluate(self, job, file_cacher):
             commands,
             job.time_limit,
             job.memory_limit,
+            max_processes,
```

Comment:
I don't think anyone's concerned with micro ops, but yes that is a bit weird, normally the pythonic way is faster...

Anyway, fixed.